packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.constant.js

```diff
@@ -53,6 +53,15 @@ export const CHART = {
     tooltips: {
       mode: 'label',
       intersect: false,
+      callbacks: {
+        title: (context) => {
+          const timeZoneOffset =
+            (new Date(context[0].label).getTimezoneOffset() / 60) * -1;
```

Comment:
Are we sure `context[0].label` will always be a valid Date ?
If not it would result in displaying NaN in the tooltip

Reply:
Yes we're sure it's a valid date.